frontend/app/assets/javascripts/spree/frontend/checkout.js.coffee

```diff
@@ -10,3 +10,8 @@ Spree.disableSaveOnClick = ->
 
 Spree.ready ($) ->
   Spree.Checkout = {}
+
+  terms_checkbox = ($ 'form.edit_order #accept_term_and_condition')
```

Comment:
Maybe just using the `id` will be enough - `accept_term_and_condition` sounds pretty unique, and people may decide to remove the `edit_order` class from the form.